docs/ref/contrib/postgres/fields.txt

```diff
@@ -920,3 +920,28 @@ define :class:`~django.contrib.postgres.constraints.ExclusionConstraint`. See
 docs/current/rangetypes.html#RANGETYPES-INCLUSIVITY>`_.
 
 .. _different bounds: https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-IO
+
+``Serial`` fields
+=================
+
+.. versionadded:: 5.1
+
+.. class:: SerialField(**options)
+
+    A field for storing an automatically incremented sequence of integers.
+
+    It's similar to :class:`AutoField <django.db.models.AutoField>`, but it
+    doesn't have to be a primary key.
+
+    It's a 32-bit integer that is is guaranteed to fit numbers from ``1`` to
+    ``2147483647``.
```

Comment:
```suggestion
    A 32-bit integer that is is guaranteed to fit numbers from ``1`` to
    ``2147483647``.
```
I don't know why I don't like "it's" here

Reply:
Maybe it's not necessary to include this sentence here at all. Similar to `AutoField`, since we say it's an `IntegerField`, it can be concluded it's 32-bit.

Reply:
Agreed 👍